gobblin-compaction/src/main/java/gobblin/compaction/mapreduce/avro/AvroKeyMapper.java

```diff
@@ -24,6 +24,8 @@
 import org.apache.hadoop.io.NullWritable;
 import org.apache.hadoop.mapreduce.Mapper;
 
+import gobblin.compaction.mapreduce.avro.AvroKeyDedupReducer.EVENT_COUNTER;
```

Comment:
I am not sure why this is showing up in the diff. Can you make sure that your branch is up-to-date with the master?